catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -45,21 +45,25 @@
    * @return
    * @throws CatalogTransformerException
    */
-  public static Geometry getKmlGeoFromWkt(final String wkt) throws CatalogTransformerException {
+  public static Geometry createKmlGeoFromWkt(final String wkt) throws CatalogTransformerException {
     if (StringUtils.isBlank(wkt)) {
       throw new CatalogTransformerException(
           "WKT was null or empty. Unable to preform KML Transform on Metacard.");
     }
 
-    com.vividsolutions.jts.geom.Geometry geo = readGeoFromWkt(wkt);
-    Geometry kmlGeo = createKmlGeometry(geo);
-    if (!POINT_TYPE.equals(geo.getGeometryType())) {
-      kmlGeo = addPointToKmlGeo(kmlGeo, geo.getCoordinate());
+    com.vividsolutions.jts.geom.Geometry geo = createJtsGeoFromWkt(wkt);
+    return createKmlGeoFromJtsGeo(geo);
+  }
+
+  public static Geometry addJtsGeoPointsToKmlGeo(
+      com.vividsolutions.jts.geom.Geometry jtsGeo, Geometry kmlGeo) {
+    if (!POINT_TYPE.equals(jtsGeo.getGeometryType())) {
+      kmlGeo = addJtsCoordinateToKmlGeo(kmlGeo, jtsGeo.getCoordinate());
     }
     return kmlGeo;
   }
 
-  private static Geometry createKmlGeometry(com.vividsolutions.jts.geom.Geometry jtsGeometry)
+  public static Geometry createKmlGeoFromJtsGeo(com.vividsolutions.jts.geom.Geometry jtsGeometry)
```

Comment:
✏️ Should be able to write positive & negative tests for this newly public method now.

Reply:
Added Tests